### DISCUSSION TITLE
Setting Up Auto Scaling for an EC2 Instance Group on AWS

### DISCUSSION THREAD

**Discussion by lucas-0liveira** [iusztinpaul/hands-on-llms#16] [Q&A]
I'm trying to configure Auto Scaling for an EC2 instance group on AWS. I've already created a launch configuration and an Auto Scaling group, and I've also defined the scaling policies. However, when I simulate a higher load on my application, I don't see new instances being launched. What could be causing this issue and how can I troubleshoot it?

**Comment by iusztinpaul** [comment]
Hello Iucas,

Do you have this issue scaling up the streaming pipeline or in general?

**Reply by lucas-0liveira** [reply]
yes

**Reply by iusztinpaul** [reply]
Please share more context, as we currently don't support auto-scaling, so I can't tell you what the issue is.
